docs/samples/performance-testing/README.md

```diff
@@ -0,0 +1,101 @@
+# HTTP Performance test
+
+This tutorial demonstrates performance testing for KServe inference services using HTTP. We will use [Iter8](https://iter8.tools) for generating load and validing service-level objectives (SLOs) for the inference service. [Iter8](https://iter8.tools) is an open-source Kubernetes release optimizer that makes it easy to ensure that your ML models perform well and maximize business value.
+
+![Iter8 HTTP performanc test](http.png)
+
+***
+
+> Performance testing of KServe inference services using gRPC is described [here](grpc.md). Canary testing using Prometheus metrics is described [here](../canary-testing/README.md). This tutorial focuses on performance testing of KServe inference services with HTTP endpoints. The main steps in this tutorial are:
+> 1. [Deploy an InferenceService](#deploy-an-inferenceservice)
+> 2. [Launch an Iter8 experiment](#launch-an-iter8-experiment)
+> 3. [View experiment report](#view-experiment-report)
+
+***
+
+## Deploy an InferenceService
+
+Create an InferenceService that exposes an HTTP port. The following serves the Scikit-learn [irisv2 model](https://kserve.github.io/website/0.10/modelserving/v1beta1/sklearn/v2/#deploy-with-inferenceservice):
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-irisv2"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      runtime: kserve-mlserver
+      storageUri: "gs://seldon-models/sklearn/mms/lr_model"
+EOF
+```
+
+Verify that your inference service is ready.
+
+```shell
+kubectl wait --for=condition=Ready --timeout=600s isvc/sklearn-irisv2
+```
+
+## Install Iter8 CLI
+Install the Iter8 CLI using `brew` as follows. You can also install using pre-built binaries as described [here](https://iter8.tools/0.13/getting-started/install/).
+
+```shell
+brew tap iter8-tools/iter8
+brew install iter8@0.13
+```
+
+## Launch an Iter8 experiment
+Iter8 introduces the notion of an *experiment* that makes it easy to verify that your inference service is ready, generate load for the inference, collect latency and error-related metrics, and assess SLOs for performance validation. Launch the Iter8 experiment inside the Kubernetes cluster.
+
+```shell
```

Comment:
@sriumcp Seems like the result is stored in kubernetes secrets, is there an option to save on cloud storage like S3 etc?

Reply:
Great question. This option is something we are considering in the future Iterations of `iter8`. For now, secrets are were the results are stored. Of course, there are limits to this approach due to the default `1MB` size limitations of k8s resource objects.

Reply:
@yuzisun I am happy to open a new PR for the main docs repo. But perhaps, this can be merged in master for now? I believe I addressed the minor edits (sklearn) as well as git-commit related comments you raised earlier.